Automattic/newspack-block-theme
# feat: homepage articles styles

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds custom styles to the Homepage Articles block. Please thoroughly test it, as I may have missed some combinations of settings.

Also, try using `newspack-grid` to a Columns block
* 33 / 33 / 33
* 25 / 50 / 25
* 15 / 70 / 15
* 30 / 70 (and 70 / 30)

I'd like to optimise the code but I'm not sure what's the best/easiest way so I moved the newspack-grid stuff into a bunch of mixins but I'm not sure if it makes sense.

![Screenshot 2024-02-10 at 11 33 49@2x](https://github.com/Automattic/newspack-block-theme/assets/177929/3d3b4c54-18bb-44b9-951b-480d9f610ecb)
![Screenshot 2024-02-10 at 11 35 51@2x](https://github.com/Automattic/newspack-block-theme/assets/177929/00729848-bdef-4947-9e8f-6b66be334bc8)

### How to test the changes in this Pull Request:

1. Add tons of different homepage articles blocks on a page – go crazy with the settings
2. Check how it looks with the "classic theme"
3. Switch to the block theme (stay on `trunk`)
4. Check how it looks now
5. Switch to this branch
6. Check how it looks now

### Other information:
